python/dgl/graphbolt/impl/fused_csc_sampling_graph.py

```diff
@@ -573,23 +573,28 @@ def _convert_to_sampled_subgraph(
         indices = C_sampled_subgraph.indices
         type_per_edge = C_sampled_subgraph.type_per_edge
         column = C_sampled_subgraph.original_column_node_ids
-        original_edge_ids = C_sampled_subgraph.original_edge_ids
+        sampled_edge_ids = C_sampled_subgraph.original_edge_ids
```

Comment:
Should we also rename the original_edge_ids in C_sampled_subgraph?
In the initial design the original_edge_ids are indexselected in C++ code, if you change the context of the variable, rename it as well.

Reply:
https://github.com/dmlc/dgl/pull/7703 I updated the doc string to clarify. It is indeed hard to rename.

Reply:
sampled_edge_ids -> edge_ids_in_fused_csc_sampling_graph.